docs/UserGuide.md

```diff
@@ -169,6 +169,23 @@ Examples:
 * `sort p/` returns person by ascending phone numbers `87438807`, `91031282` followed by `92492021`<br>
   ![result for 'sort p/'](images/sortByPhoneNumberResult.png)
 
+### Adding a meeting to a person: `meeting`
+
+Add a meeting to a person based on specified description and date.
+
+Format: `meeting INDEX [d/DESCRIPTION] [s/DATETIME]`
+
+* Adds a meeting to the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3,…​ and tally within range index of the displayed list.
+* At both of the fields must be provided and valid.
```

Comment:
nit: Should this just be `Both of the fields` without the "at"? Perhaps the latter part can also be edited: `and be valid values.`